.github/actions/clean-vpc/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"context"
+	"log"
+	"os"
+
+	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/actions/cloud"
+	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/actions/networkpeer"
+)
+
+func main() {
+	err := setGCPCredentials()
+	if err != nil {
+		log.Fatal(err)
+	}
+	gcpOk, err := deleteGCPVPCBySubstr(cloud.GoogleProjectID, networkpeer.GCPVPCName)
```

Comment:
Where is `GCPVPCName` value coming from?

Reply:
This value is taken from network peering tests: "github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/actions/networkpeer"